### DISCUSSION TITLE
Automatically kill/remove fake players when two or more real players join the game

### DISCUSSION THREAD

**Discussion by swishkin** [gnembon/fabric-carpet#1609] [Q&A]
I'm looking for a way to make the fake players I use to AFK farms automatically disappear if two or more players join the game, to allow for the best possible performance for real players while utilizing fake players to AFK for resources when one or no players are online.

I think a command block might be able to accomplish this, but I'm curious if carpet itself, or scarpet, would be able to produce this effect.

Any ideas?

**Comment by James103** [comment]
This line should suffice for the check itself, but I have not tested it:
```
length(filter(player('all'), _~'player_type' != 'fake')) >= 2
```
Change the `2` at the end to the number of non-fake players you want to have online for the check to trigger (return true).

To kill all fake players, start with the following (also untested):
```
for(player('all'), if(_~'player_type' == 'fake', modify(_, 'kill')));
```
